users.go

```diff
@@ -582,3 +582,113 @@ func (s *UsersService) DeleteEmailForUser(uid int, eid int, options ...OptionFun
 
 	return s.client.Do(req, nil)
 }
+
+// ImpersonationToken represents an ImpersonationToken.
+//
+// GitLab API docs: https://docs.gitlab.com/ce/api/users.html#get-all-impersonation-tokens-of-a-user
+type ImpersonationToken struct {
+	ID        int        `json:"id"`
+	Active    *bool      `json:"active"`
+	Token     *string    `json:"token"`
+	Scopes    *[]string  `json:"scopes"`
+	Revoked   *bool      `json:"revoked"`
+	Name      *string    `json:"name"`
+	CreatedAt *time.Time `json:"created_at"`
+	ExpiresAt *time.Time `json:"expires_at"`
+}
+
+// GetAllImpersonationTokensOptions represents the available GetAllImpersonationTokens() options.
+//
+// GitLab API docs:
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/9-0-stable/doc/api/users.md#get-all-impersonation-tokens-of-a-user
+type GetAllImpersonationTokensOptions struct {
+	State *string `url:"state,omitempty" json:"state,omitempty"`
+}
+
+// Get all impersonation tokens of a user.
+//
+// GitLab API docs:
+// https://gitlab.com/gitlab-org/gitlab-ce/blob/9-0-stable/doc/api/users.md#get-all-impersonation-tokens-of-a-user
+func (s *UsersService) GetAllImpersonationTokens(uid int, opt *GetAllImpersonationTokensOptions, options ...OptionFunc) ([]*ImpersonationToken, *Response, error) {
```

Comment:
I understand what it stands for :wink: But if you look at https://github.com/xanzy/go-gitlab/blob/master/groups.go and see the fucnction signatures, then maybe you see my point.

I use the `gid` and `pid` which are always an `interface{}` when the API expects either a `string` or an `int`. When it only expects an `int` I write the thing it expects the ID from. So either `project` of `user`. So to keep things consistent within this package, please use `user` instead of `uid` and `token` instead of `itid`.

Thanks!